applications/GeoMechanicsApplication/custom_elements/README.md

```diff
@@ -127,7 +227,8 @@ density of water $\rho^w$ $[\mathrm {kg/m^3}]$ is a function of temperature and
 
 $$ \rho^w = 9.998396 \cdot 10^2 + 6.764771  \cdot 10^{-2} \cdot T - 8.993699  \cdot 10^{-3} \cdot T^2 + 9.143518 \cdot 10^{-5} \cdot T^3 - 8.907391 \cdot 10^{-7} \cdot T^4 + 5.291959  \cdot 10^{-9} \cdot T^5 - 1.359813  \cdot 10^{-11} \cdot T^6 $$
 
-Note: the dependency of water density and viscosity to temperature is user defined and the user can turn this feature on or off in the JSON file.
```

Comment:
Also note that line 224 produces an incorrect closing bracket ] to a ( is this intended ?

![image](https://github.com/user-attachments/assets/c06c7d35-4f5f-44ab-9900-3ed0a6c447af)

Reply:
It is fixed